CHANGES/909.bugfix.rst

```diff
@@ -1,3 +1,9 @@
 Revert to using the public argument parsing API
 :c:func:`PyArg_ParseTupleAndKeywords` under Python 3.13
```

Comment:
Since this change note hasn't made it into an actual release, I think it's best to just edit it in place
```suggestion
:c:func:`PyArg_ParseTupleAndKeywords` under Python 3.12
```